src/helper/helper.js

```diff
@@ -405,20 +401,22 @@ function isString_(value) {
  * @private
  */
 function merge_(from, to) {
+  const preventMerge = from['_clear'];
   for (const property in from) {
     if (hasOwn(from, property)) {
       const fromProperty = from[property];
-      if (isArray_(fromProperty)) {
+      if (isArray_(fromProperty) && !preventMerge) {
```

Comment:
Optional, but for readability of the condition statements I usually prefer naming boolean variables so they don't need a not prefix, especially when the variable name makes that format a double negative as here. 

So for example, this could be e.g. `const allowMerge = !from['_clear']` and `if (isArray() && allowMerge) // ...`.  This changes how the condition gets read from "If the property is an array and merging *is not prevented*" to "If the property is an array and merging *is allowed*".

Reply:
Thank you for the readability pointers :) Updated!